rest_framework_simple_api_key/admin.py

```diff
@@ -0,0 +1,47 @@
+import typing
+
+from django.contrib import admin, messages
+from django.http.request import HttpRequest
+
+from .models import APIKey
+
+
+class ApiKeyAdmin(admin.ModelAdmin):
+    list_display = ("entity", "revoked", "expires_at")
+
+    list_filter = ("entity", "revoked", "expires_at")
+
+    def get_readonly_fields(
+        self, request: HttpRequest, obj: APIKey = None
+    ) -> typing.Tuple[str, ...]:
+        fields = ()
+
+        if obj and obj.revoked:
+            fields += ("name", "revoked", "expires_at")
+
+        return fields
+
+    def save_model(
+        self,
+        request: HttpRequest,
+        obj: APIKey,
+        form: typing.Any = None,
+        change: bool = False,
+    ) -> None:
+
+        if not obj.pk:
```

Comment:
Why `if not obj.pk`? What are you trying to do? Explain, please.

Reply:
I'll add better comments then. The goal is: if the object is been created, then we need to display the `api_key` value.